added documentation for delete self endpoint of doord service (#43)
doordeck/docs: source/includes/_account.md

Parameter | Required | Description
--------- | ------- | -----------
displayName | true | User's desired display name

## Delete Account

Deletes the currently logged in user's account, associated with the application the user is logged in.

> <br>
> <b>Warning! </b>This operation is irreversible!

Reviewer: We tend to use the warning/aside system, it makes a big red box:
```suggestion
<aside class="warning">
This operation is irreversible!
</aside>
```

```shell
curl 'https://api.doordeck.com/account' \
   -X DELETE \

>   --header 'Content-Type: application/json' \

Reviewer: ```suggestion
```

Don't think you need a content type

  --header 'Authorization: Bearer TOKEN' \
```

### HTTP Request

`DELETE https://api.doordeck.com/account`

> 
> ### Request Parameters
> 
> This request has no body/ parameters. The applicationId and userId are taken from the authentication token.

Reviewer: ```suggestion
```

I think we can just get rid of this section.
